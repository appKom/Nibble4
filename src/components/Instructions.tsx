import React, { FC } from "react";
import styled from "styled-components";
import Card from "atoms/Card";
import { OnlineBlue } from "utility/style";

const Instructions: FC = () => (
  <Card>
    <Container>
      <h1>Har du ikke bruker?</h1>
      <p>
        For å bruke Online sitt kiosksystem, Nibble, så må man ha registrert
        sitt NTNU-adgangskort på sin Online-bruker. Har du ikke bruker? Følg
        stegene beskrevet nedenfor.
      </p>
      <ol>
        <li>Scan ditt NTNU brukerkort på venstre side</li>
        <li>
          Logg inn med din <span>online.ntnu.no</span> innloggingsinformasjon.
        </li>
        <ul>
          <li>Du må snu maskinen og bruke maskinens tastatur </li>
        </ul>
        <li>
          Kortet ditt vil nå være registrert på din bruker, og du kan nå logge
          inn på Nibble. Du kan se dette under ‘min side’ på{" "}
          <span>online.ntnu.no/profile</span>
        </li>
        <li>
          For å legge til NOK på brukeren din må du inn på{" "}
          <span>online.ntnu.no/payments/wallet</span>. Den siden kan du også
          finne ved din å trykke på saldo’ fra nedtrekksmenyen ved å trykke på
          profilen din på <span>online.ntnu.no</span>
        </li>
      </ol>
    </Container>
  </Card>
);

const Container = styled.div`
  margin: auto;
  h1 {
    text-align: center;
  }
  span {
    color: ${OnlineBlue};
    text-decoration: underline;
  }
  p:first-of-type {
    padding: 1rem 3rem;
    color: #5d6455;
  }
  ol {
    li {
      margin-top: 1rem;
    }
  }
`;

export default Instructions;
