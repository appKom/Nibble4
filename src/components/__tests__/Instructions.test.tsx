import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Instructions from "../Instructions";

afterEach(cleanup);
it("renders", () => {
  const { asFragment } = render(<Instructions />);
  expect(asFragment()).toMatchSnapshot();
});
