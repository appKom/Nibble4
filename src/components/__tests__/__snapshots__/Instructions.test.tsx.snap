// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ bIzbNl"
  >
    <div
      class="sc-gsTCUz zfIbE"
    >
      <h1>
        Har du ikke bruker?
      </h1>
      <p>
        For å bruke Online sitt kiosksystem, Nibble, så må man ha registrert sitt NTNU-adgangskort på sin Online-bruker. Har du ikke bruker? Følg stegene beskrevet nedenfor.
      </p>
      <ol>
        <li>
          Scan ditt NTNU brukerkort på venstre side
        </li>
        <li>
          Logg inn med din 
          <span>
            online.ntnu.no
          </span>
           innloggingsinformasjon.
        </li>
        <ul>
          <li>
            Du må snu maskinen og bruke maskinens tastatur 
          </li>
        </ul>
        <li>
          Kortet ditt vil nå være registrert på din bruker, og du kan nå logge inn på Nibble. Du kan se dette under ‘min side’ på 
          <span>
            online.ntnu.no/profile
          </span>
        </li>
        <li>
          For å legge til NOK på brukeren din må du inn på 
          <span>
            online.ntnu.no/payments/wallet
          </span>
          . Den siden kan du også finne ved din å trykke på saldo’ fra nedtrekksmenyen ved å trykke på profilen din på 
          <span>
            online.ntnu.no
          </span>
        </li>
      </ol>
    </div>
  </div>
</DocumentFragment>
`;
